lib/ProductOpener/Display.pm

```diff
@@ -4870,7 +4870,7 @@ sub search_and_display_products ($request_ref, $query_ref, $sort_by, $limit, $pa
 
 		$request_ref->{structured_response} = {
 			page => $page,
-			page_size => $limit,
+			page_size => 0 + $limit,
```

Comment:
we should put 0+ to page and skip as well

Reply:
I tried that and it didn't seem to help on page. I'll investigate separately when I have a bit more time